dashboard/app/models/user.rb

```diff
@@ -1822,10 +1822,15 @@ def self.initialize_new_oauth_user(user, auth, params)
     user.provider = auth.provider
     user.uid = auth.uid
     user.name = name_from_omniauth auth.info.name
-    user.family_name = auth.info.family_name if auth.info.family_name.present?
-    user.user_type = params['user_type'] || auth.info.user_type
+    user.user_type = params['user_type'] || params[:user_type] || auth.info.user_type
```

Comment:
Are you adding `params[:user_type]` as a symbol because you are creating it in the sign-up flow. 
So now new users should have `params[:user_type]` and old users will have `params['user_type']`, is that a correct interpretation?
Could we just have `params['user_type']`? I'm asking for my own understanding.